aspnetcore/performance/caching/response.md

```diff
@@ -31,6 +31,11 @@ The primary HTTP header used for caching is `Cache-Control`. See the [HTTP 1.1 C
 
 The web server can cache responses by adding the response caching middleware. See [Response caching middleware](middleware.md) for more information.
 
+## Distributed Cache Tag Helper
+
+The [Distributed Cache Tag Helper](xref:mvc/views/tag-helpers/builtin-th/DistributedCacheTagHelper) enables distributed caching.
```

Comment:
cc @javiercn I think this is fine but since you are more familiar with it, do you want to add anything else here?